src/citrine/resources/file_link.py

```diff
@@ -251,11 +259,21 @@ def _upload_file(file_path: str, uploader: _Uploader):
             The input uploader object with its s3_version field now populated.
 
         """
+
+        additional_s3_opts = {
+            'use_ssl': uploader.s3_use_ssl,
+            'config': Config(s3={'addressing_style': uploader.s3_addressing_style})
+        }
+
+        if uploader.s3_endpoint_url:
```

Comment:
Are you looking for `s3_endpoint_url` to be `True` or a string or simply not None?

Reply:
Oh good point - this should be:

if ... != None: